app/search_builders/curation_concerns/filter_suppressed_with_roles.rb

```diff
@@ -0,0 +1,27 @@
+module CurationConcerns
+  module FilterSuppressedWithRoles
+    extend CurationConcerns::FilterSuppressed
```

Comment:
So my idea of this was, creating an object that inherits the methods from `CurationConcerns::FilterSuppressed` and overrides them to cut down on code duplication. Would you just include the module instead of extend the module?